teachable.rb

```diff
@@ -0,0 +1,31 @@
+module  Teachable
```

Comment:
I would expect `Teachable` to be an attribute of a student, not a teacher.  Also, each teacher class *is* a `Teacher`.  Since you can say that, better to give them a superclass.  Teachers are also people.  How can you structure your inheritance hierarchy?